### ISSUE TITLE
[1.7.10] Crash on startup

### ISSUE THREAD

**Issue by GedeonGrays** [HenryLoenwind/middletorch#8]
Hello o/

**The Middle Torch version:** 1.0.8
**Crash Log:** https://pastebin.com/CVfUBVHK

The previous version (1.0.4) works properly ¯\_(ツ)_/¯

**Comment by HenryLoenwind** [comment]
I think I see what the issue with the jar file is, but I'm a bit puzzled by it---I'm using the same file in a big modpack.

I'll need to dive into that ForgeGradle fork I had to use to compile for 1.7.10. Seems they changed more than making it work with the changed URLs and newer gradle versions.

**Comment by HenryLoenwind** [comment]
Okay, I've found the cause of this. It seems that when I launch the game from the build environment, it builds an unobfuscated version of the jar file and overwrites the release version with it. And I did run the game once between building the final file and uploading it. After I copied it into the modpack I'm currently playing to test it there...

Fixed version 1.0.9 has been uploaded. Thanks for notifying me about this.
